itou/scripts/management/commands/export_asp_approval_data.py

```diff
@@ -1,88 +1,163 @@
+import argparse
 import csv
-from io import StringIO
-from os.path import splitext
-
-import django.db.models as models
+import datetime
 
 from itou.approvals.models import Approval
+from itou.users.enums import UserKind
 from itou.utils.command import BaseCommand
 
 
-CSV_SEPARATOR = ";"
-FILENAME_SUFFIX = "_updated"
-
-
-class ItouHashError(Exception):
-    pass
+def search(data, queries, *, step=0):
+    if len(queries) <= step:
+        return step, Approval.objects.none()
+    if queries[step] - set(data.keys()) == set():
```

Comment:
nit, sans grande conviction.
```suggestion
    if queries[step] <= set(data.keys()):
```

Reply:
C'est l'intention initial donc bien mieux. :grinning:.